src/scripts/main.js

```diff
@@ -101,12 +112,13 @@ function populateSignatures(data, force) {
 		message.dataset.ftsig = '1';
 
 		const spinner = new Spinner(message);
+
+		const oldSigs = [...message.querySelectorAll('div[href="http://ftsig"]')];
```

Comment:
Could this be Array.from? It's a little easier to understand what's going on at a glance imo.